tests/test_stl_binders.py

```diff
@@ -314,22 +314,29 @@ def test_map_delitem():
 def test_map_view_types():
     map_string_double = m.MapStringDouble()
     unordered_map_string_double = m.UnorderedMapStringDouble()
+    map_string_float = m.MapStringFloat()
+    unordered_map_string_float = m.UnorderedMapStringFloat()
     map_string_double_const = m.MapStringDoubleConst()
     unordered_map_string_double_const = m.UnorderedMapStringDoubleConst()
 
-    assert map_string_double.keys().__class__.__name__ == "KeysView[str]"
-    assert map_string_double.values().__class__.__name__ == "ValuesView[float]"
-    assert map_string_double.items().__class__.__name__ == "ItemsView[str, float]"
+    assert map_string_double.values().__class__.__name__ == "ValuesView[double]"
+    assert map_string_float.values().__class__.__name__ == "ValuesView[float]"
```

Comment:
Please keep the tests for `keys()` and `items()` to make sure that those are also named as expected